beacon-chain/core/helpers/committee_test.go

```diff
@@ -879,6 +880,15 @@ func TestCurrentEpochSyncSubcommitteeIndices_UsingCommittee(t *testing.T) {
 	index, err := CurrentEpochSyncSubcommitteeIndices(state, 0)
 	require.NoError(t, err)
 	require.DeepEqual(t, []uint64{0}, index)
+
+	root, err := syncPeriodBoundaryRoot(state)
+	require.NoError(t, err)
+
+	// Test that cache was able to fill on miss.
```

Comment:
From the test, how can you be sure this was a cache miss?

Reply:
Added a condition above to validate cache was empty. Thanks.